www_src/components/snackbar/snackbar.jsx

```diff
@@ -0,0 +1,31 @@
+var React = require('react/addons');
+
+var Snackbar = React.createClass({
+  getInitialState: function() {
+    return {
+      message: this.props.message,
+      hidden: true
+    };
+  },
+  render: function () {
+    var className = "snackbar";
+    if (this.state.hidden) {
+      className = "hidden " + className;
+    }
+    return <div className={className}>{ this.state.message }</div>;
+  },
+  showMessage: function(msg) {
+    this.setState({
+      message: msg,
+      hidden: false
+    }, function() {
+      setTimeout(() => {
```

Comment:
ideally this isn't a timeout, but the "3 seconds" needs to be hard-coded somewhere, and it seems more sensible to do so on the JS side than putting it in the CSS and the listening for transition events to change the state.